cpp/tests/utilities/cudf_test_utils.cuh

```diff
@@ -36,6 +38,38 @@
 using gdf_col_pointer = typename std::unique_ptr<gdf_column, 
                                                  std::function<void(gdf_column*)>>;
 
+
+// Prints a vector and valids
+template <typename T>
+void print_vector_and_valid(T * v,
+                        gdf_valid_type * valid,
+                        const size_t num_rows)
+{
+  auto functor = [&valid, &v](int index) -> std::string {
+    if (gdf_is_valid(valid, index))
+      return std::to_string((int)v[index]);
+    return std::string("@");
+  };
+  std::vector<int> indexes(num_rows);
+  std::iota(std::begin(indexes), std::end(indexes), 0);
+  std::transform(indexes.begin(), indexes.end(), std::ostream_iterator<std::string>(std::cout, ", "), functor);
+  std::cout << std::endl;
+}
+
+static constexpr int ValidSize = 32;
+using ValidType = uint32_t;
+
+static size_t  valid_size(size_t column_length)
```

Comment:
This is a redefinition of `get_num_chars_bitmask`. https://github.com/rapidsai/cudf/blob/master/cpp/src/utilities/cudf_utils.h#L21